### ISSUE TITLE
Add commands for Transfer endpoint manager APIs

### ISSUE THREAD

**Issue by sirosen** [globus/globus-cli#291]
We're working through the last of endpoint manager / "management console" ( https://github.com/globus/globus-sdk-python/milestone/11 ) support in the SDK now.

It would be nice to have a CLI version with commands for all of these activities.
Command structure/names TBD.

`globus admin` or `globus endpoint admin` have a good feel, as they connote the user->endpoint or user->task relationship.
i.e. `globus admin task pause` and `globus admin task cancel` "make sense"

We should also consider `globus endpoint manager`, `globus manage`, and the related families of command names.

Having the activities delineated like the rest of the CLI seems good for consistency. That means using the operative entity like `endpoint`, `task`, `bookmark`, and `endpoint server` as an intermediate command name, and the verb, `show`, `create`, `delete`, as a leaf command.
So, `globus admin task pause` is good, but `globus admin pause task` is bad.

Currently blocked on upcoming SDK release.

**Comment by aaschaer** [comment]
I prefer `globus admin` as a naming convention.

`globus endpoint admin` could be confusing for the handful of `endpoint_manager` methods that don't actually need an endpoint (although they all allow filtering by endpoint if we want to be explicit about what you are an admin of)

`globus manage` would be good, but again a handful of `endpoint_manager` methods only require you to have a monitor rather than manager role, and admin nicely covers both of those cases.

**Comment by mikkel14** [comment]
I'm looking at the endpoint management functions added in the milestone @sirosen mentioned above. Can anyone point out any endpoint manager activities I am missing below? Going through the milestone, I was unsure if some of the functionality was already exposed through other commands. Assuming we follow the `globus admin` naming convention, this could be something like the following:

Pause rule management activities
- `globus admin pause show`
- `globus admin pause create`
- `globus admin pause update`
- `globus admin pause delete`

Task management activities
- `globus admin task pause`
- `globus admin task cancel` 
- `globus admin task resume`

**Comment by sirosen** [comment]
This stalled out but I'm marking it as a feature to target for 3.0.
Especially now that we have a rebuilt testing framework, it should be possible to add minimal versions of the desired commands and test them.

Even though it's more verbose than the suggested names from a few years back, I think `globus endpoint admin` would be the best choice. Aaron's point about there being APIs which don't require an endpoint aside, it clearly namespaces the commands and indicates that they are Transfer-related.

**Comment by sirosen** [comment]
I've moved this item into the Globus internal issue tracker to try to better get it slotted in amongst our priorities. I know it's been open for ages, but I would really like us to get this done at some point.

Because the GitHub issue is a duplicate for our team, I'm closing it as part of some GitHub issues spring cleaning.

**Comment by kabuhr** [comment]
How's this coming?  We have a use case where we'd like to have someone with the "activity_monitor" role audit guest collection permissions (with read-only access to the ACL list). We can write a custom Python script to access this information, but it would be nice if it was available via the standard CLI tool.

**Comment by sirosen** [comment]
I'm sorry to share that we never got back around to this work item!

It is possible to do some of this activity via the `globus api transfer` subcommands, which are new since the last touch on this issue. You would have to reference the Transfer API documentation to formulate the commands, but it should be possible.

In the years since this was filed, the CLI has evolved quite significantly, and it has become clear that `globus admin` would not be an appropriate namespace for these commands. I have ideas about what changes we need to support this, but we unfortunately haven't had the resources (personnel/time-wise) to pursue those plans.

**Comment by kabuhr** [comment]
Got it. I had somehow missed the whole `globus api` interface. Thanks for that!
